kolibri/plugins/learn/assets/src/views/LearningActivityBar.vue

```diff
@@ -453,6 +455,10 @@
         message: 'View folder resources',
         context: 'Tooltip text.',
       },
+      contentMenuLabel: {
+        message: 'Content page menu label',
```

Comment:
**Content renderers**

I'm a bit torn about this ``nav`` element, as it's not really a navigation bar but almost a toolbar, right? I'm wondering if we would be misplacing the user expectation by announcing it's a navigation... If we are keeping it, we can eliminate the ``aria-label`` (again, not ``arialabel``), because there's only one on the page.

![Ubuntu18 (start)  Running  - Oracle VM VirtualBox_018](https://user-images.githubusercontent.com/1457929/186934051-9e9d47ae-103a-405b-865e-674851e99d77.png)